README.md

```diff
@@ -80,6 +83,7 @@ Then, initialize the NX monorepo with the following command
 ```bash
 ./nx init
```

Comment:
this is not an initialization command, and this step is not required: whenever you use nx, it bootstraps/syncs itself.